scripts/run_backend_tests.py

```diff
@@ -648,7 +716,8 @@ def check_coverage(
     cmd = [
         sys.executable, '-m', 'coverage', 'report',
          '--omit="%s*","third_party/*","/usr/share/*"'
-         % common.OPPIA_TOOLS_DIR, '--show-missing']
+         % common.OPPIA_TOOLS_DIR, '--show-missing',
+         '--skip-covered']
```

Comment:
When you run coverage with this command, does the resulting output automatically skip 100% lines? (What does it look like, can you show an example?)

If it already skips 100% lines then do you need all the complicated formatting still? Or does putting a message at the end suffice?

Reply:
@seanlip i have included the screenshots of lines with formatting and without formatting.
**with formatting :** 
![image](https://github.com/oppia/oppia/assets/97088265/e946df89-f32f-41d9-b4bb-9ba53ddaf1ae)
![image](https://github.com/oppia/oppia/assets/97088265/f425d9fd-5472-4ec5-b932-99f2dbd96717)

**without formatting:** 
![image](https://github.com/oppia/oppia/assets/97088265/7eff184e-d490-4777-b7df-4ade0762835b)

Reply:
Thanks for the screenshots. I agree the formatting looks a bit nicer, but typically we won't run into this case where so many lines are uncovered, it'll be just one or two files. 

Remember the original point of this issue -- show only the files that miss coverage, make clear what specific actions the dev should take next. The first part has been solved by the changed command. So just do what's needed to achieve the second part -- I don't think the complexity of the code needed for the pretty-printing is worth it.

Reply:
ok @seanlip i will remove the formatting and then update the commit.

Reply:
@seanlip i removed the code that was formatting the report and updated the test for the condition.